broadinstitute/pearl
# AR-76 adding skeleton admin UI

In order to start working on datstat question importers, and also start putting skeleton UX for portal admin together, this starts the ui-admin module.  The frame was ported from the rara repo, and stripped of everything except a login page.  I've created a new, even simpler "unauthedLogin" route that we can use to just say "here's an email, log me in as that" and be trusted.  Obviously, this UX and endpoint will be disabled/removed in production.  

Along the way, I made a new SetupPopulator which handles everything needed for a fresh install, including environments and adminUsers.  I also migrated some of the services to extend CrudService.  

TO TEST:
1. run PopulateCliApp with args "setup seed"
2. confirm 2 admin users, breilly and dbush, exist in your database
3. in the `ui-admin` directory, run `npm start`
4. go to localhost:8080
5. confirm you can log in with `dbush@broadinstitute.org`. and you can't log in with an email not in your db
6.  you should see this amazing ui after you log in successfully
![image](https://user-images.githubusercontent.com/2800795/207644529-34849d17-0c0b-40e0-a869-19b135289e35.png)
